travel_platform/smart_ordering.py

```diff
@@ -22,24 +22,41 @@ def smart_ordering(data_dict, filter_by, order_by='ASC', limit=100):
     Returns:
         list: Sorted list of dictionaries with applied filters.
     '''
+    order_by = order_by.upper()
+    # print filter_by, order_by, limit
     if order_by == 'DESC':
         sorted_data = sorted(data_dict,
                              key=lambda note: (filter_by not in note, note.get(filter_by)),
                              reverse=True)
     else:
         sorted_data = sorted(data_dict,
                              key=lambda note: (filter_by not in note, note.get(filter_by)))
+    if filter_by == None:
```

Comment:
bad condition.
it should be `if filter_by is None` or `if not filter_by`

Also it would be great to have this condition in the beginning